config/dispatcher/dispatcher.ex

```diff
@@ -67,18 +70,51 @@ defmodule Dispatcher do
     Proxy.forward conn, [], "http://document-versions/agendaitems/" <> id <> "/pieces/restore"
   end
 
+
+  ### Agenda comparison
+
   get "/agendas/:agenda_id/compare/:compared_agenda_id/agenda-items", @json_service do
-    Proxy.forward conn, [], "http://agenda-sort/agendas/" <> agenda_id <> "/compare/" <> compared_agenda_id <> "/agenda-items"
+    Proxy.forward conn, [], "http://agenda-comparison/agendas/" <> agenda_id <> "/compare/" <> compared_agenda_id <> "/agenda-items"
   end
 
   get "/agendas/:agenda_id/compare/:compared_agenda_id/agenda-item/:agenda_item_id/pieces", @json_service do
-    Proxy.forward conn, [], "http://agenda-sort/agendas/" <> agenda_id <> "/compare/" <> compared_agenda_id <> "/agenda-item/" <> agenda_item_id <> "/documents"
+    Proxy.forward conn, [], "http://agenda-comparison/agendas/" <> agenda_id <> "/compare/" <> compared_agenda_id <> "/agenda-item/" <> agenda_item_id <> "/documents"
   end
 
   post "/agendas/:id/agendaitems/pieces/files/archive", @json_service do
     Proxy.forward conn, [], "http://file-bundling-job-creation/agendas/" <> id <> "/agendaitems/documents/files/archive"
   end
 
+
+  ### Agenda approval and meeting management
+
+  post "/meetings/:meeting_id/reopen", @json_service do
+    Proxy.forward conn, [], "http://agenda-approve/meetings/" <> meeting_id <> "/reopen"
+  end
+
+  post "/meetings/:meeting_id/close", @json_service do
+    Proxy.forward conn, [], "http://agenda-approve/meetings/" <> meeting_id <> "/close"
+  end
+
+  post "/agendas/:agenda_id/approve", @json_service do
+    Proxy.forward conn, [], "http://agenda-approve/agendas/" <> agenda_id <> "/approve"
+  end
+
+  post "/agendas/:agenda_id/reopen", @json_service do
+    Proxy.forward conn, [], "http://agenda-approve/agendas/" <> agenda_id <> "/reopen"
+  end
+
+  post "/agendas/:agenda_id/close", @json_service do
+    Proxy.forward conn, [], "http://agenda-approve/agendas/" <> agenda_id <> "/close"
+  end
+
+  delete "/agendas/:agenda_id", @json_service do
```

Comment:
Dit betekent in praktijk dat wat de `DELETE` betreft, de agendas-call nooit meer naar die hieronder (http://cache/agendas/) zal doorvallen. Ook `agenda.destroyRecord()` komt hier uit. Da's ok? Ik denk het wel, maar wilde er toch nog even op wijzen, mocht dat onbewust gepasseerd zijn.